Docs - Hamlet Learn (#99)
hamlet-io/docs: docs/learn/layers/intro.md

---
sidebar_label: overview
title: Hamlet Layers
---

import Admonition from 'react-admonitions';

> As an organisation grows more and more people want to have a say in the way that things are done. It's alright for them to talk the talk, but we know who the actual implementation of it all is going to fall down to - the engineers. 

Reviewer: This feels kinda clunky, I'm not sure it really covers what hamlet does or problems people might be having

Author: Yeah I should have written the "intro" last rather than first. Will re-write this bit.

Managing all of these requirements, together with their conflicts and edge cases is very challenging. Hamlet was designed to address this challenge.

> ## A Layer-ed Approach

Reviewer: ```suggestion
## A Layered Approach
```

> Hamlet captures a wholistic view of an organsations requirements, and applies each one across its entire scope. A particular security policy may be applied for just a single product, or across the entire organsation. The implementation of that policy is the same either way, and in both cases it is only implemented the once - but the layer at which it is defined ensures it will apply across the scope it is intended.

Reviewer: ```suggestion
Hamlet captures a holistic view of an organisations infrastructure deployment requirements, and applies them across its entire scope. A particular security policy may be applied for just a single product, or across the entire organisation. The implementation of that policy is the same either way, and in both cases it is only implemented the once - but the layer at which it is defined ensures it will apply across the scope it is intended.
```
